docker-compose.yml

```diff
@@ -26,7 +26,7 @@ services:
         - elixir-node-4001-db:/elixir_node/apps/aecore/priv/rox_db_4001
       stdin_open: true
       tty: true
-      command: -e "pubkey = Client.get_info(\"elixir-node-4000:4000\") |> elem(1) |> Map.get(:public_key)" -e "decoded_pubkey = Account.base58c_decode(pubkey)" -e "peer = Map.put(%{}, :host, 'elixir-node-4000') |> Map.put(:port, 3015) |> Map.put(:pubkey, decoded_pubkey)" -e "Aecore.Peers.Worker.try_connect(peer)" -e "Aecore.Miner.Worker.resume()"
+      command: -e "pubkey = Aehttpclient.Client.get_info(\"elixir-node-4000:4000\") |> elem(1) |> Map.get(:public_key)" -e "decoded_pubkey = Aecore.Account.Account.base58c_decode(pubkey)" -e "peer = Map.put(%{}, :host, 'elixir-node-4000') |> Map.put(:port, 3015) |> Map.put(:pubkey, decoded_pubkey)" -e "Aecore.Peers.Worker.try_connect(peer)" -e "Aecore.Miner.Worker.resume()"
```

Comment:
Does it work? Because when I tested docker-compose, I found out that it doesn't memorize the variables if you pass them from one -e command to another, so you can't initialize for example pubkey variable, and after that to use its value in another -e command, tell me if I am wrong, because I am not sure about that.

Reply:
I couldn't run the docker since some dependency fails to compile, I haven't tested it so I need someone else to run it, I'll change it to one combined command and push, if you could test it it would be great.